### DISCUSSION TITLE
Unrpyc and Termux

### DISCUSSION THREAD

**Discussion by Veldzix** [termux/termux-app#3898] [Q&A] [answered]
I've been trying to run Unrpyc with no luck
I've been able to use unrpa to extract rpa files
But every time I manage to install Unrpyc
It goes to the next line with 
">"
And I can't do anything short of exiting Termux

Would be nice to have a straight forward step by step guide on how to operate Unrpyc on Termux because I just don't get it

**Comment by Veldzix** [comment, marked as answer]
I figured it out 
Download the zip for Unrpyc

Copy the directory of Unrpyc zip 

In Termux enter
pip install (paste the directory)

When installed
Use it by copying the directory of the game folder
Then entering in Termux 
unrpyc.py (paste the directory)

**Comment by SASUKE444ITACHI777** [comment]
> ter um

How did you install rpa in termux mn?

**Reply by Veldzix** [reply]
you search for unrpa here on github and download the zip

with the latest Termux

use
termux-setup-storage

then
pkg upgrade

then 
pkg install python

then
pkg install python-pip

next copy the directory of the unrpa zip

then use 
pip install (paste the directory)

to use unrpa
copy the directory of the folder that contains the rpa files 

then use
cd  (paste the directory)

then use
unrpa *.rpa 
you can replace the * with the name of a specific rpa file to only decompile that 
otherwise its a full sweep

follow up with
rm *.rpa
this removes all rpa files
if you targeted a specific rpa then do the same here

**Comment by SASUKE444ITACHI777** [comment]
> Estou tentando executar o Unrpyc sem sorte. Consegui usar o unrpa para extrair os arquivos rpa. Mas toda vez que consigo instalar o Unrpyc, ele vai para a próxima linha com ">" E não consigo fazer nada além de sair do Termux.
> 
> Seria bom ter um guia passo a passo direto sobre como operar o Unrpyc no Termux porque eu simplesmente não entendo

I have tried to extract the file several times but it does not extract, help me out
